components/Asana-Card/AsanaCard.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function AsanaCard({ name, img }) {
+  return (
+    <StyledArticle>
+      <ImageWrapper>
+        <Image
+          src={img}
+          alt={name}
+          width={1000}
+          height={1000}
+          layout="responsive"
+        />
+      </ImageWrapper>
+      <h2>{name}</h2>
+    </StyledArticle>
+  );
+}
+
+const ImageWrapper = styled.section`
+  width: 100px;
+  height: 100px;
+`;
+
+const StyledArticle = styled.article`
+  border-radius: 22px;
+  box-shadow: 0px 0px 4px rgba(93, 107, 234, 0.42);
+  display: flex;
+  gap: 3rem;
+  justify-content: center;
+  padding: 1.5rem 3rem 1.5rem 3rem;
+  width: 60vw;
```

Comment:
Please don't use `vw` for this, you can use a padding on the parent element + a max-width instead